apps/builder/app/builder/features/sidebar-left/panels/pages/settings.tsx

```diff
@@ -771,13 +775,50 @@ const deletePage = (pageId: Page["id"]) => {
   });
 };
 
+const duplicatePage = (pageId: Page["id"]) => {
+  const pages = pagesStore.get();
+  const page =
+    pages?.homePage.id === pageId
+      ? pages.homePage
+      : pages?.pages.find((page) => page.id === pageId);
+  if (page === undefined) {
+    return;
+  }
+
+  const newPageId = nanoid();
+  const { name = page.name, indexString } =
+    // parse "name" or "name (indexString)"
```

Comment:
what is this thing? why do we ave such page names? :)

Reply:
It's auto added suffix with copy number